packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -1539,6 +1539,44 @@ describe('TableView', function () {
         expect(document.activeElement).toHaveTextContent('Bar');
         expect(body.scrollTop).toBe(1000);
       });
+
+      describe('Space key with focus on a link within a cell', () => {
+        it('should toggle selection and prevent scrolling of the table', () => {
```

Comment:
I test to verify that the link within the first row in the Table is has the same label: `expect(link.textContent).toBe('Foo 1');`, but this may not explicitly verify that the default has been prevented.

Reply:
I've added a test for `preventDefault` being called on `fireEvent.keyDown`/`keyUp`, and to verify that the table body scrollTop position remains unchanged.